lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -380,16 +380,29 @@ def add_check_manifest_lock_script_phase
             phase = TargetIntegrator.create_or_update_build_phase(native_target, BUILD_PHASE_PREFIX + phase_name)
             native_target.build_phases.unshift(phase).uniq! unless native_target.build_phases.first == phase
             phase.shell_script = <<-SH.strip_heredoc
-              diff "${PODS_PODFILE_DIR_PATH}/Podfile.lock" "${PODS_ROOT}/Manifest.lock" > /dev/null
-              if [ $? != 0 ] ; then
-                  # print error to STDERR
-                  echo "error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation." >&2
-                  exit 1
+              set -e
```

Comment:
Please can you use long names for these?

Reply:
I didn't know there were long names 😬
Do you also want me to change the other scripts that @keith recently updated?

Reply:
Doesn't need to be in the same pr, but it would be nice to do a sweep of them. In scripts it's always preferrable to use the verbose names for everything imo.